backend/inspirehep/pidstore/minters/bai.py

```diff
@@ -4,14 +4,52 @@
 #
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
-
+from flask import current_app
 from inspire_utils.record import get_values_for_schema
 
 from inspirehep.pidstore.minters.base import Minter
+from inspirehep.pidstore.providers import InspireExternalIdProvider
+from inspirehep.pidstore.providers.bai import InspireBAIProvider
 
 
 class BAIMinter(Minter):
     pid_type = "bai"
+    provider = InspireBAIProvider
+
+    def __init__(self, object_uuid, data):
+        super().__init__(object_uuid, data)
+        if not current_app.config.get("FEATURE_FLAG_ENABLE_BAI_PROVIDER", False):
```

Comment:
Please add the feature flag in the https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/config.py#L23 section

Reply:
:+1: I will also add Feature Flag for Texkey as I forgot to do it before. (Still it's not necessary for the code but I know that it's useful to see all feature flags in one place)